tests/data/dataconverter/readers/mpes/config_file.json

```diff
@@ -14,328 +14,399 @@
+    "incident_energy": "@attrs:metadata/instrument/beam/pump/incident_energy",
+    "incident_energy/@units": "eV",
+    "incident_energy_spread": "@attrs:metadata/instrument/beam/pump/incident_energy_spread",
+    "incident_energy_spread/@units": "eV",
+    "incident_wavelength": "@attrs:metadata/instrument/beam/pump/incident_wavelength",
+    "incident_wavelength/@units": "nm",
+    "pulse_duration": "@attrs:metadata/instrument/beam/pump/pulse_duration",
+    "pulse_duration/@units": "fs",
+    "incident_polarization": "@attrs:metadata/instrument/beam/pump/incident_polarization",
+    "incident_polarization/@units": "V^2/mm^2",
+    "pulse_energy": "@attrs:metadata/instrument/beam/pump/pulse_energy",
+    "pulse_energy/@units": "µJ",
+    "average_power": "@attrs:metadata/instrument/beam/pump/average_power",
+    "average_power/@units": "mW",
+    "extent": "@attrs:metadata/instrument/beam/pump/extent",
+    "extent/@units": "µm",
+    "fluence": "@attrs:metadata/instrument/beam/pump/fluence",
+    "fluence/@units": "mJ/cm^2",
+    "associated_source": "/entry/instrument/source_pump"
+  },
+  "/ENTRY[entry]/INSTRUMENT[instrument]/MANIPULATOR[manipulator]": {
+    "temperature_sensor": {
+      "name": "sample_temperature",
+      "measurement": "temperature",
+      "value": "@attrs:metadata/file/trARPES:Carving:TEMP_RBV",
+      "value/@units": "K"
+    },
+    "sample_bias_voltmeter": {
+      "name": "sample_bias",
+      "measurement": "voltage",
+      "value": "@attrs:metadata/file/KTOF:Lens:Sample:V",
+      "value/@units": "V"
+    },
+
+    "depends_on": "/entry/instrument/manipulator/transformations/trans_z",
+    "TRANSFORMATIONS[transformations]": {
+      "AXISNAME[trans_z]": -0.32,
+      "AXISNAME[trans_z]/@depends_on": "rot_z",
+      "AXISNAME[trans_z]/@transformation_type": "translation",
+      "AXISNAME[trans_z]/@units": "m",
+      "AXISNAME[trans_z]/@vector": [
+        0,
+        0,
+        1
+      ],
+      "AXISNAME[rot_z]/@depends_on": "rot_x",
+      "AXISNAME[rot_z]": -25.0,
+      "AXISNAME[rot_z]/@transformation_type": "rotation",
+      "AXISNAME[rot_z]/@units": "degrees",
+      "AXISNAME[rot_z]/@vector": [
+        0,
+        0,
+        1
+      ],
+      "AXISNAME[rot_x]/@depends_on": ".",
+      "AXISNAME[rot_x]": -90.0,
+      "AXISNAME[rot_x]/@transformation_type": "rotation",
+      "AXISNAME[rot_x]/@units": "degrees",
+      "AXISNAME[rot_x]/@vector": [
+        1,
+        0,
+        0
+      ]
+    }
+  },
+  "/ENTRY[entry]/SAMPLE[sample]": {
+    "preparation_date": "@attrs:metadata/sample/preparation_date",
+    "sample_history/notes": "@attrs:metadata/sample/sample_history",
+    "chemical_formula": "@attrs:metadata/sample/chemical_formula",
+    "description": "@attrs:metadata/sample/chemical_formula",
+    "name": "@attrs:metadata/sample/chemical_formula",
+    "situation": "vacuum",
+    "SUBSTANCE[substance]/molecular_formula_hill": "@link:/entry/sample/chemical_formula",

```

Comment:
Do we need both entries? Is not the upper one "depricated"?

Reply:
Yes, I don't know why I did it like this here. I changed this to only use `molecular_formula_hill`.